modules/calib3d/src/chessboard.cpp

```diff
@@ -3373,7 +3373,7 @@ cv::Scalar Chessboard::Board::calcEdgeSharpness(cv::InputArray _img,float rise_d
     }
     if(count == 0)
     {
-        std::cout  <<"calcEdgeSharpness: checkerboard too small for calculation." << std::endl;
+        // std::cout  <<"calcEdgeSharpness: checkerboard too small for calculation." << std::endl;
```

Comment:
Using `CV_LOG_DEBUG()` is preferable for such messages.

Reply:
I commented it out for consistency with the other messages in the file. The return value already tells us what we need to know.

Reply:
@asmorkalov  Your does look much cleaner. My main objection is that now I have to turn off logging, but that's not a real problem :)